src/main.ts

```diff
@@ -27,6 +27,9 @@ import App from './App.vue'
 import { initLdClient, fetchConfig } from '@/utils'
 import KeycloakService from 'sbc-common-components/src/services/keycloak.services'
 
+// get rid of "element implicitly has an 'any' type..."
+declare const window: any
```

Comment:
This is a neat trick and is needed because of this code:
```
window['sentryDsn']
```

I guess TS is smart enough to know that `sentryDsn` isn't a (normal) property of `window`.